packages/browser/src/integrations/dedupe.ts

```diff
@@ -27,6 +28,7 @@ export class Dedupe implements Integration {
         // Juuust in case something goes wrong
         try {
           if (self._shouldDropEvent(currentEvent, self._previousEvent)) {
+            logger.warn(`Event dropped due to being a duplicate of previously captured event.`);
```

Comment:
Wait should we do this with https://github.com/getsentry/sentry-javascript/blob/master/packages/integrations/src/dedupe.ts as well?